app/src/main/java/com/tolkiana/tacoapp/services/ProductsService.kt

```diff
@@ -0,0 +1,39 @@
+package com.tolkiana.tacoapp.services
+
+import com.github.kittinunf.fuel.httpGet
+import com.tolkiana.tacoapp.Product
+import com.github.kittinunf.result.Result
+import com.github.kittinunf.result.getAs
+import com.tolkiana.tacoapp.utilities.DataParser
+
+/**
+ * Created by tolkiana on 11/10/17.
+ */
+class ProductsService(val url: String) {
```

Comment:
I think this `ProductService` class is a great idea. It abstracts away getting data into functions that take a callback, and that's how I would build out this API data fetching class too. 👍